MonoGame.Framework/Audio/SoundEffectInstance.OpenAL.cs

```diff
@@ -260,14 +308,44 @@ private void PlatformSetVolume(float value)
 
         internal void PlatformSetReverbMix(float mix)
         {
+#if !MONOMAC
+            if (!OpenALSoundController.Efx.IsInitialized)
+                return;
+            reverb = mix;
+#endif
         }
 
         internal void PlatformSetFilter(FilterMode mode, float filterQ, float frequency)
         {
+#if !MONOMAC
+            if (!OpenALSoundController.Efx.IsInitialized)
+                return;
+
+            applyFilter = true;
+            switch (mode) {
+            case FilterMode.BandPass:
+                filterType = EfxFilterType.Bandpass;
+                break;
+                case FilterMode.LowPass:
+                filterType = EfxFilterType.Lowpass;
+                break;
+                case FilterMode.HighPass:
+                filterType = EfxFilterType.Highpass;
+                break;
+            }
+            this.filterQ = filterQ;
+            this.frequency = frequency;
+#endif
```

Comment:
I think you have a misunderstanding of the API here.  `PlatformSetFilter` expects immediate application of the filter settings if the sound instance is playing.

You should be able to play a long running or looping sound effect and modify both the filter and reverb settings interactively without restarting playback.